### PR TITLE
Retrieve condition

### PR DESCRIPTION
**Pull request by vickyjaiswal0308**
-> Created a new Branch
-> Updated sushi-config.yaml
-> Created a flow diagram and gave it a reference in .md file

### CODE REVIEW

**Review by declankieran-nhsd — CHANGES_REQUESTED** [review]
The diagram is good, but I think we should change it a bit, so that the look and feel is consistent between the diagrams.  We should probably agree on a palette of draw.io componets to use for things like timeline, alt fragments etc.

Also (and I think I need to update my PR as I think I missed a place), I think referring to it as Condition is too tight a scope and we should all it an RA record, unless we want to be specifying this at the individual endpoint level, which we might do later, but this seems to be better scoped at the RA record, so I think changing the initial interaction from `Retrieve Condition Request` to `Retrieve RA Record` makes more sense.

I'll attach a version with the changes I'm thinking

**Comment by declankieran-nhsd** [comment]
Diagram changes suggested

File: input/pagecontent/retrieve-condition.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Summary
+
+<p>Practioner wishes to read/retrieve Condition information</p>
```
Comment (declankieran-nhsd) [inline input/pagecontent/retrieve-condition.md:3]: Change this from 

<p>Practioner wishes to read/retrieve Condition information</p> 

to

<p>Practioner wishes to read/retrieve RA Record</p> 

Just to keep consistent as mentioned in previous comment

**Comment by vickyjaiswal0308** [comment]
> Diagram changes suggested (the Example 1 note also follows whats in the other diagram, and would tie to the query output I have going in the other branch.
> 
> retrieve-condition drawio

Done the 

> Diagram changes suggested (the Example 1 note also follows whats in the other diagram, and would tie to the query output I have going in the other branch.
> 
> retrieve-condition drawio

Done this change

File: input/pagecontent/retrieve-condition.md (hunk 1 above)
Comment (vickyjaiswal0308) [reply input/pagecontent/retrieve-condition.md]: Done this change

**Comment by vickyjaiswal0308** [comment]
> Diagram changes suggested (the Example 1 note also follows whats in the other diagram, and would tie to the query output I have going in the other branch.
> 
> retrieve-condition drawio

Done this change

**Review by declankieran-nhsd — APPROVED** [review]

**Pull request merged**